ShareX.UploadersLib/Forms/UploadersConfigForm.Designer.cs

```diff
@@ -2527,6 +2574,267 @@ private void InitializeComponent()
+            this.lvSeafileLibraries.Name = "lvSeafileLibraries";
+            this.lvSeafileLibraries.UseCompatibleStateImageBehavior = false;
+            this.lvSeafileLibraries.View = System.Windows.Forms.View.Details;
+            this.lvSeafileLibraries.SelectedIndexChanged += new System.EventHandler(this.lvSeafileLibraries_SelectedIndexChanged);
+            // 
+            // colSeafileLibraryName
+            // 
+            resources.ApplyResources(this.colSeafileLibraryName, "colSeafileLibraryName");
+            // 
+            // colSeafileLibrarySize
+            // 
+            resources.ApplyResources(this.colSeafileLibrarySize, "colSeafileLibrarySize");
+            // 
+            // colSeafileLibraryEncrypted
+            // 
+            resources.ApplyResources(this.colSeafileLibraryEncrypted, "colSeafileLibraryEncrypted");
+            // 
+            // btnSeafilePathValidate
+            // 
+            resources.ApplyResources(this.btnSeafilePathValidate, "btnSeafilePathValidate");
+            this.btnSeafilePathValidate.Name = "btnSeafilePathValidate";
+            this.btnSeafilePathValidate.UseVisualStyleBackColor = true;
+            this.btnSeafilePathValidate.Click += new System.EventHandler(this.btnSeafilePathValidate_Click);
+            // 
+            // txtSeafileDirectoryPath
+            // 
+            resources.ApplyResources(this.txtSeafileDirectoryPath, "txtSeafileDirectoryPath");
+            this.txtSeafileDirectoryPath.Name = "txtSeafileDirectoryPath";
+            this.txtSeafileDirectoryPath.TextChanged += new System.EventHandler(this.txtSeafileDirectoryPath_TextChanged);
+            // 
+            // lblSeafileWritePermNotif
+            // 
+            resources.ApplyResources(this.lblSeafileWritePermNotif, "lblSeafileWritePermNotif");
+            this.lblSeafileWritePermNotif.Name = "lblSeafileWritePermNotif";
+            // 
+            // lblSeafilePath
+            // 
+            resources.ApplyResources(this.lblSeafilePath, "lblSeafilePath");
+            this.lblSeafilePath.Name = "lblSeafilePath";
+            // 
+            // txtSeafileUploadLocationRefresh
+            // 
+            resources.ApplyResources(this.txtSeafileUploadLocationRefresh, "txtSeafileUploadLocationRefresh");
+            this.txtSeafileUploadLocationRefresh.Name = "txtSeafileUploadLocationRefresh";
+            this.txtSeafileUploadLocationRefresh.UseVisualStyleBackColor = true;
+            this.txtSeafileUploadLocationRefresh.Click += new System.EventHandler(this.txtSeafileUploadLocationRefresh_Click);
+            // 
+            // lblSeafileSelectLibrary
+            // 
+            resources.ApplyResources(this.lblSeafileSelectLibrary, "lblSeafileSelectLibrary");
+            this.lblSeafileSelectLibrary.Name = "lblSeafileSelectLibrary";
+            // 
+            // grpSeafileAccInfo
+            // 
+            this.grpSeafileAccInfo.Controls.Add(this.myListView1);
```

Comment:
I was gonna say I didn't know what those were.  I should have looked harder
for remnants like that.
On Oct 13, 2015 2:18 AM, "Jaex" <notifications@github.com> wrote:

> In ShareX.UploadersLib/Forms/UploadersConfigForm.Designer.cs
> <https://github.com/ShareX/ShareX/pull/1061#discussion_r41830262>:
>
> > +            //
> > +            // txtSeafileUploadLocationRefresh
> > +            //
> > +            resources.ApplyResources(this.txtSeafileUploadLocationRefresh, "txtSeafileUploadLocationRefresh");
> > +            this.txtSeafileUploadLocationRefresh.Name = "txtSeafileUploadLocationRefresh";
> > +            this.txtSeafileUploadLocationRefresh.UseVisualStyleBackColor = true;
> > +            this.txtSeafileUploadLocationRefresh.Click += new System.EventHandler(this.txtSeafileUploadLocationRefresh_Click);
> > +            //
> > +            // lblSeafileSelectLibrary
> > +            //
> > +            resources.ApplyResources(this.lblSeafileSelectLibrary, "lblSeafileSelectLibrary");
> > +            this.lblSeafileSelectLibrary.Name = "lblSeafileSelectLibrary";
> > +            //
> > +            // grpSeafileAccInfo
> > +            //
> > +            this.grpSeafileAccInfo.Controls.Add(this.myListView1);
>
> I can't find this in UI.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ShareX/ShareX/pull/1061/files#r41830262>.
>